External/ProcessEx.cs

```diff
@@ -308,6 +308,42 @@ public void SetString(PointerEx absoluteAddress, string Value)
             SetArray(absoluteAddress, Value.Bytes());
         }
 
+        public PointerEx OffsetCalculator(string moduleName, int baseOffset, int[] offsets)
+        {
+            PointerEx moduleAddr = GetLoadedModuleAddress(moduleName);
```

Comment:
The function should be updated to not need a module name. 

Example proposed usage: result = FollowPointerArray(game["module_name"] + offset, new byte[]{ 0x1, 0x2, 0x3 });